shared-bin/src/fairings.rs

```diff
@@ -11,16 +11,20 @@ pub fn log_launch() -> impl Fairing {
                 rocket.config().port
             );
 
-            tracing::info!(endpoint = %http_endpoint, "HTTP interface is ready");
+            tracing::info!(target: "http", endpoint = %http_endpoint, "HTTP interface is ready");
         })
     })
 }
 
 /// Attach this fairing to enable logging Rocket HTTP requests
 pub fn log_requests() -> impl Fairing {
-    AdHoc::on_request("Rocket HTTP request", |request, _data| {
+    AdHoc::on_response("Log status code for request", |request, response| {
```

Comment:
nice one! I didn't notice that fairing that gives the status code!

nit: you might want to adjust the comment and perhaps the name to make it clear that it also logs response codes :)

Reply:
> nit: you might want to adjust the comment and perhaps the name to make it clear that it also logs response codes :)

Yeah, the current one is not wrong per so, just not as complete as it could be. I don't think it matters much so I'll leave it!